cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,113 @@
 /// <reference types='cypress' />
+const { faker } = require('@faker-js/faker');
+let user = {};
+const userEdit = {
+  lastName: faker.person.lastName(),
+  email: faker.internet.email(),
+  department: faker.commerce.department()
+};
+
+beforeEach(() => {
+  cy.task('generateUser').then((generateUser) => {
+    user = generateUser;
+  });
+  cy.visit('/webtables');
+});
 
 describe('Web Tables page', () => {
-  it('', () => {
+  it('shold check pagination', () => {
+    cy.get('select').select('5 rows');
+
+    cy.createWorker(user, 3);
+
+    cy.get('.-totalPages').should('contain', '2');
```

Comment:
add actions:
click on the button next and previous and assert that current page changes